backend/service/github/github.go

```diff
@@ -361,6 +361,20 @@ func (s *svc) CompareCommits(ctx context.Context, ref *RemoteRef, compareSHA str
 	}, nil
 }
 
+/*
+ * CommitRange() is used to get multiple commits between 2 SHAs. It is useful because we can do one API call and get a bunch
+ * of commits, rather than calling GetCommit() multiple times.
+ * compareSHA is the base (parent) while the ref.ref is the head
+ */
+func (s *svc) CommitRange(ctx context.Context, baseSHA string, ref *RemoteRef) ([]*githubv3.RepositoryCommit, error) {
```

Comment:
should we just add this data to CompareCommits?

Reply:
idk, should we? I guess the reason why I didn't was because I wanted to make it like other helper functions (GetCommit)  that didn't have protos but rather just the struct definitions in the file.  What should be the standard way moving forward? (protos, or not protos?) 

I wasn't sure if I should take over the `scgithubv1.CommitComparison` proto message.  If I put it in there I would need the github commit proto definiton (or recreate some version of it) which I thought was awkward.

Reply:
I realized after digging that its actually used in the envoydeploys module which seems to not be used anymore (was made by smercatoris who has since left).  Should we delete all of that as well?